Wikipedia/Code/MWKHistoryList.m

```diff
@@ -84,6 +84,12 @@ - (nullable WMFArticle *)mostRecentEntry {
     return [[self.dataStore.viewContext executeFetchRequest:request error:nil] firstObject];
 }
 
+- (nullable WMFArticle *)mostRecentEntryInManagedObjectContext:(NSManagedObjectContext *)context {
```

Comment:
The method above could now call this method with `self.dataStore.viewContext` but not a required change